ow_lander/scripts/utils.py

```diff
@@ -32,7 +38,13 @@ def start_traj_recording(delete_prev_traj,bagname):
   command = "rosbag record -O " + bagname + " /planning/joint_states"
   p = subprocess.Popen(command, stdin=subprocess.PIPE, shell=True, cwd='.')
 
-def stop_traj_recording(result, bagname):
+def stop_traj_recording(result,       # type: bool
+                        bagname):     # type: str
+
+  print "utils>stop_traj_recording"
+  print type(result)
+  print type(bagname)
+
```

Comment:
Are all of these for debugging the types
```suggestion
```

Reply:
Yes, before writing the type I printed out the type to double check. I'll remove those